docs/index.md

```diff
@@ -1,6 +1,6 @@
-# Conveyal Transit Data Tools
+# IBI Transit Data Tools
```

Comment:
Suggest changing to IBI TRANSIT-data-tools if that will be our long term branding for this

Reply:
I'm not sure if that has been finalized yet. @jon-campbell-ibigroup was advocating pretty heavily for TRANSIT-wrench. 😉

Maybe once we decide on that we can just submit another PR because I think the exact name/branding could be replaced in a couple of other spots as well.

Reply:
Sounds good